_ols-1/week14.md

```diff
@@ -7,49 +7,49 @@ title: Week 14 - Meet your mentor + Group 1 - Final Presentation Rehearsal
 - [Week 14-1: Mentor meeting 6](#week-14-1--mentor-meeting-6)
 - [Week 14-2: Group 1 - Final presentation rehearsal](#week-14-2--group-1---final-presentation-rehearsal)
 
-# Week 14-1: Mentor meeting 6
+## Week 14-1: Mentor meeting 6
 
 **Date**: Week of April 20, 2020
 
 **Duration**: 30 minutes
```

Comment:
hmmm, 30 mins or 90?

Reply:
30 mins for the mentoring calls, the cohort call is below with 90 mins. Should we mention that on the top?

Reply:
nah, that was me being inattentive! Should be okay I think :)